lib/logstash/outputs/lumberjack.rb

```diff
@@ -73,6 +74,10 @@ def flush(events, close = false)
       retry
     end # begin
   end
+
+  def close
+    buffer_flush(:force => true) 
```

Comment:
conceptually they should be the same, but this would prevent sending the event to the codec.